tools/loader/ImageClassifier.cpp

```diff
@@ -99,9 +106,19 @@ void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
   std::tie(imgHeight, imgWidth, isGray) = getPngInfo(filenames[0].c_str());
   const size_t numChannels = isGray ? 1 : 3;
 
+  if(imageformat != "NHWC") {
   // N x C x H x W
-  result->reset(ElemKind::FloatTy,
+    result->reset(ElemKind::FloatTy,
                 {numImages, numChannels, imgHeight, imgWidth});
+  }
+  //N x H x W x C
```

Comment:
please move comment inside else branch, to match style of previous comment

Reply:
Please do not assume that anything that is not NCHW is automatically NHWC. We may add more formats in the future. So, e.g. assert inside else that it is NHWC

Reply:
Will do